CAP/gap/DiscreteSkeletalFiniteCategory.gi

```diff
@@ -0,0 +1,192 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# CAP: Categories, Algorithms, Programming
+#
+# Implementations
+#
+
+####################################
+#
+# Constructor
+#
+####################################
+
+##
+InstallGlobalFunction( DiscreteSkeletalFiniteCategory,
```

Comment:
A discrete category is automatically skeletal. The adjective skeletal should be removed from the name.

Reply:
I see, my suggestion was "evil". So keep everything as is.